platform/platform-filter-delegate/src/test/java/org/codice/ddf/platform/filter/delegate/DelegateServletFilterTest.java

```diff
@@ -329,6 +327,11 @@ public void testRemoveSecurityFilter() throws IOException, ServletException {
     }
 
     @Test
+    public void testRemoveNullSecurityFilter() throws IOException, ServletException {
+        delegateServletFilter.removeSecurityFilter(null);
```

Comment:
If you verify one of those methods isn't called, then it would show that the added logic is working as intended while adding an assertion to the unit test.  My comment is nit picky though